janitor/tasks/labware_location/main.py

```diff
@@ -13,10 +18,15 @@ def sync_changes_from_labwhere(config):
     logger.info("Starting sync labware locations task...")
     db_labwhere = Database(config.LABWHERE_DB)
     db_mlwh = Database(config.MLWH_DB)
-    sync_job_interval = config.SYNC_JOB_INTERVAL_SEC + config.SYNC_JOB_OVERLAP_SEC
+
+    latest_timestamp = db_mlwh.execute_query(GET_LATEST_TIMESTAMP_QUERY, {})[0][0]
```

Comment:
I don't know if there are any situations where `execute_query` might return `None` or the first `[0]` might return `None` because those would likely raise exceptions.  In that case you might want to wrap this in a `try: except:` block and catch `TypeError` which is thrown when you try to index `None`.  In that case we probably need to log the problem, abort the sync task and wait for it to come around again.